### ISSUE TITLE
Bug: Template strings not being extracted when using self-closing tags

### ISSUE THREAD

**Issue by aleesaan** [jsverse/transloco-keys-manager#155]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is this a regression?

No

### Current behavior

[Angular 15.1.0](https://github.com/angular/angular/releases/tag/15.1.0) has introduced [self-closing tags for custom elements](https://github.com/angular/angular/pull/48535), but when trying to extract keys from a template using self-closing tags, no keys get extracted.

### Expected behavior

Keys should be extracted also in files using self-closing tags.

### Please provide a link to a minimal reproduction of the bug

-

### Please provide the environment you discovered this bug in

```markdown
Transloco: 4.2.6
Transloco Keys Manager: 3.7.0
Angular: 15.1.x
Node: 16
Package Manager: 8
OS:
```

### Additional context
Simply updating the `@angular/compiler` to >=15.1 should do the trick, as it will correctly parse the nodes for files with self-closing custom tags.

### I would like to make a pull request for this bug

Yes

**Comment by shaharkazaz** [comment]
@aleesaan This is a big issue since this also requires moving this package to ESM instead of commonjs.
Would you like to take a shot at this task? I started but didn't find the time to finish the task.

**Comment by aleesaan** [comment]
The next two weeks will be tight for me as well, I'm not sure I can pick this up sorry :(

**Comment by shaharkazaz** [comment]
@aleesaan It's ok, it will wait until someone will have the time to contribute 🙂

**Comment by krechtern** [comment]
We are also facing this issue.

**Comment by shaharkazaz** [comment]
@Ben305 I don't have the time to get to it, I tried starting but it's quite a lot of work.
Anyone who is willing is welcome to open a PR 🙂

**Comment by shaharkazaz** [comment]
Should be resolved by #180
